charts/handshake-node/values.yaml

```diff
@@ -118,3 +118,27 @@ storage:
 affinity: {}
 nodeSelector: {}
 tolerations: []
+
+# Prometheus exporter configuration
+prometheus:
+  enabled: true
+  image:
+    repository: blinklabs/handshake-prometheus-exporter
+    tag: 0.0.1
+    pullPolicy: IfNotPresent
+  refreshSeconds: 1
+  banAddressMetrics: true
+  logLevel: "INFO"
+  port: 12000
+  resources:
```

Comment:
I would use `resources: {}` with this set commented to give the user a nice recommendation without enforcing resource limits on the user.